.github/workflows/zxcron-extended-test-suite.yaml

```
@@ -107,9 +105,10 @@ jobs:
             echo "xts-tag-commit=${XTS_COMMIT}" >> $GITHUB_OUTPUT
             echo "xts-tag-commit-author=${AUTHOR_NAME} <${AUTHOR_EMAIL}>" >> $GITHUB_OUTPUT
             echo "xts-tag-commit-email=${AUTHOR_EMAIL}" >> $GITHUB_OUTPUT
-            echo "interim-commit-list=${CLEAN_COMMIT_LIST}" >> $GITHUB_OUTPUT
             echo "### XTS-Candidate commit found" >>  $GITHUB_STEP_SUMMARY
             echo "xts-tag-commit=${XTS_COMMIT}" >> $GITHUB_STEP_SUMMARY
+            echo "### Commit List" >> ${GITHUB_STEP_SUMMARY}
```

```suggestion
            echo "### Commit List" >> "${GITHUB_STEP_SUMMARY}"
```

Should be double quoted ideally to prevent globbing in the event the path contains spaces or colons.